try {
    let a=9;
    let b=0;
    let c=a/b
    console.log(c)

}
catch {
    console.error('this is error get always error')
}
finally{
    console.error('this may excuted or may not?')
    
}